### ISSUE TITLE
[Accessibility] New messages button are not read when focus on it

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3203]
## Version

`master`

## Describe the bug

When we <kbd>TAB</kbd> on the "New messages" button, no content is read.

This is because the button parent has `role="separator"`, and it is has a special flag called "presentational children". That means, all descendants will have `role="presentation"` automatically assigned to it and cannot be overridden.

We need to turn the separator into a button instead. This is supported as long as we provide all required interactions.

## Steps to reproduce

1. Type `proactive card`
1. Immediately type `help`, scroll up a bit
1. Wait for the "New messages" button to show up
1. Press <kbd>TAB</kbd> to focus on the "New messages" button

## Expected behavior

AT will read "New messages button" or similar phrases.

Instead, AT speak nothing.

## Additional context

[Bug]
